sound/soc/sof/intel/mtl.c

```diff
@@ -256,7 +256,7 @@ static int mtl_dsp_pre_fw_run(struct snd_sof_dev *sdev)
 		dev_err(sdev->dev, "failed to power up gated DSP domain\n");
 
 	/* make sure SoundWire is not power-gated */
-	snd_sof_dsp_update_bits(sdev, HDA_DSP_HDA_BAR, MTL_HFPWRCTL,
+	snd_sof_dsp_update_bits(sdev, HDA_DSP_BAR, MTL_HFPWRCTL,
```

Comment:
The change is correct, but the commit message needs to be updated.

It's not just a matter of consistency, at the hardware level the HDA_BAR power registers shadow the DSP_BAR.

Reply:
> The change is correct, but the commit message needs to be updated.
> 
> It's not just a matter of consistency, at the hardware level the HDA_BAR power registers shadow the DSP_BAR.
Ack, will update the commit message.

Reply:
the commit message is now even more confusing.

suggested edit: The hfPRWRCTL registers in the HDA BAR shadow the values of the same registers in the DSP BAR, so let's modify the latter - as done already for other accesses.

Reply:
It's all "Host Power Management / Clock Control (ULP) Registers"